.github/workflows/deploy.yml

```diff
@@ -94,11 +94,11 @@ jobs:
               --set ENABLE_HTTPS=true \
               --set PLATFORM_NAME='Open edx $VERSION Demo'
           "
-      # - name: Install extra xblocks/packages
-      #   run: |
-      #     $SSH "#! /bin/bash -e
-      #       $TUTOR config save --set 'OPENEDX_EXTRA_PIP_REQUIREMENTS=[\"edx-event-routing-backends>=5.2.1,<6.0.0\"]'
-      #     "
+      - name: Install extra xblocks/packages
+        run: |
+          $SSH "#! /bin/bash -e
+            $TUTOR config save --set 'OPENEDX_EXTRA_PIP_REQUIREMENTS=[\"edx-event-routing-backends>=5.3.0,<6.0.0\"]'
```

Comment:
Note that you can now write:

    $TUTOR config save --append OPENEDX_EXTRA_PIP_REQUIREMENTS="edx-event-routing-backends>=5.3.0,<6.0.0"